src/config.rs

```diff
@@ -645,6 +672,10 @@ mod tests {
             "another_password"
         );
         assert!(config.users[1].current_user);
+        assert_eq!(
+            config.users[1].last_used,
+            Some(Utc.with_ymd_and_hms(2007, 10, 19, 7, 23, 4).unwrap())
```

Comment:
The build is failing, because `Utc.with_ymd_and_hms` requires that you `use chrono::TimeZone`